doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md

```diff
@@ -2,126 +2,112 @@
-                    {
-                        "contextElement" : {
-                        "attributes" : [
-                            {
-                                "name" : "speed",
-                                "type" : "float",
-                                "value" : "112.9",
-                                "metadatas": []
-                            },
-                            {
-                                "name" : "oil_level",
-                                "type" : "float",
-                                "value" : "74.6",
-                                "metadatas": []
-                            }
-                        ],
-                        "type" : "car",
-                        "isPattern" : "false",
-                        "id" : "car1"
-                    },
-                    "statusCode" : {
-                        "code" : "200",
-                        "reasonPhrase" : "OK"
-                    }
-                ]
-            }
-        }
-    }
-
-As said, Flume events are not much more different than the above representation: there is a set of headers and a body. This is an advantage, since allows for a quick translation between formats. The equivalent <i>object representation</i> (not any real data format) for such a notified NGSI event could be the following Flume event:
-
-    flume-event={
-        headers={
-	         timestamp=1429535775,
-	         transaction-id=0123456789ABCDEF,
-	         fiware-correlator=ABCDEF1234567890,
-	         fiware-service=vehicles,
-	         fiware-servicepath=/4wheels,
-	         notified-entities=car1_car,
-	         grouped-entities=cars,
-	         grouped-servicepath=/mycars
-        },
-        body={
-	         entityId=car1,
-	         entityType=car,
-	         attributes=[
-	             {
-	                  attrName=speed,
-	                  attrType=float,
-	                  attrValue=112.9
-	             },
-	             {
-	                  attrName=oil_level,
-	                  attrType=float,
-	                  attrValue=74.6
-	             }
-	         ]
-	     }
-    }
+###<a name="section1.2"></a>Example
+Let's assume the following not-intercepted event regarding a received notification (the code below is an <i>object representation</i>, not any real data format):
+
+```
+notification={
+   headers={
+	   fiware-service=hotel1,
+	   fiware-servicepath=/other,/suites,
+	   correlation-id=1234567890-0000-1234567890
+   },
+   body={
+      {
+	      entityId=suite.12,
+	      entityType=room,
+	      attributes=[
+	         ...
+	      ]
+	   },
+	   {
+	      entityId=other.9,
+	      entityType=room,
+	      attributes=[
+	         ...
+	      ]
+	   }
+	}
+}
+```
+
+As can be seen, two entities (`suite.12` and `other.9`) of the same type (`room`) within the same FIWARE service (`hotel`) but different service paths (`/suites` and `/other`) are notified. `NGSIRestHandler` will crate two `NGSIEvent`'s:
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1308/commits/7dbe4ab1ef9d2cc343af43c0e0fe9c6a960dc4af